recipes/cvxportfolio/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "cvxportfolio" %}
+{% set version = "0.0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # no sdist provided, no tags on github
```

Comment:
Please add the URL for the issue upastream.

Reply:
Can you address this comment?

Reply:
It was already added before your last changes